### ISSUE TITLE
move perennial/data/wrappers to chipper

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#680]
Here is a TODO in `copySupplementalPhetioFiles.js`:

```

  // Load the master list of all wrappers
  // TODO: Should this be in a file? Would we be duplicating perennial? We want a reproducible build
  let wrappers = [
    'phet-io-wrappers/active',
    'phet-io-wrappers/event-log',
    'phet-io-wrappers/index',
    'phet-io-wrappers/studio',
    'phet-io-wrappers/login',
    'phet-io-wrappers/mirror-inputs',
    'phet-io-wrappers/record',
    'phet-io-wrappers/playback',
    'phet-io-wrappers/screenshot',
    'phet-io-wrappers/state',
    'phet-io-wrappers/wrapper-template',
    'phet-io-wrapper-classroom-activity',
    'phet-io-wrapper-lab-book'
  ];
```

Why don't we just move this over to chipper, so that we have reproducible builds?

**Comment by jonathanolson** [comment]
Hmm, is it OK for perennial to (in an automated fashion) update a file in chipper?

Additionally, we'd want to decide whether we want it in two places (might be fine), or just in chipper (phetmarks will break for slightly-older sims).

**Comment by zepumph** [comment]
>Additionally, we'd want to decide whether we want it in two places (might be fine), or just in chipper (phetmarks will break for slightly-older sims).

I think that it breaks phetmarks to not have this list in chipper, because when you checkout old shas with n-2 wrappers in it, you are still using the master perennial file. 

What purpose does this info being timeless serve? 
Benefits for chipper:
* builds know exactly what wrappers we support for the wrapper suite
* phetmarks know exactly what wrappers are in phet-io-wrappers

**Comment by samreid** [comment]
I'm confused because copySupplementalPhetioFiles.js is already in chipper, can you please clarify?

**Comment by zepumph** [comment]
I think this should be done before stable api release.

**Comment by zepumph** [comment]
>Maybe it is worth dev meeting discussion to see if there is a problem with auto-committing on chipper?

By this do you mean a script that will take the perennial list (that phetmarks uses) and commit it to chipper (which uses it on build) every so often? If so I think this is the best idea yet, as long as it is well documented. Marking for dev meeting, even though I will not be attending the next onw.

I

**Comment by samreid** [comment]
Can phetmarks use a list defined in chipper instead of perennial?  It seems the list of wrappers must be in chipper so it will "version" with the rest of the code.

**Comment by jonathanolson** [comment]
> Can phetmarks use a list defined in chipper instead of perennial?

Yes, but it has been annoying checking out older sims and not being able to use phetmarks. Should we think about backwards-compatibility with it?

**Comment by samreid** [comment]
Perhaps the natural solution is to check out correspondingly older phetmarks?  Should we record its SHAs?

**Comment by jonathanolson** [comment]
> Perhaps the natural solution is to check out correspondingly older phetmarks? Should we record its SHAs?

Could consider it, I'm not sure.

**Comment by zepumph** [comment]
I'm ok not supporting phetmarks when older sims are checked out. For the most part we can just go to the index wrapper to navigate to what is needed. 

(not advocating for this) The other option would probably be like you said, to keep phetmarks as part of the dependencies, so that it is always in sync with the phet-io wrappers/chipper shas. In my mind the index wrapper is already filling that niche.

**Comment by zepumph** [comment]
At dev meeting today, we decided that we are ok with phetmarks not working when we check out old shas.

From here we will move the wrappers list from perennial to chipper, so they aren't replicated and we will still have reproducible builds

**Comment by zepumph** [comment]
Implemented above, @samreid will you please give it a spot check. Usages include login wrapper and phetmarks.

**Comment by samreid** [comment]
I reviewed the commits and everything seems reasonable.  It also made me realize we should be testing the login wrapper, at least for RCs.  I'll create a new issue for that.  @zepumph OK to close this issue?
